### DISCUSSION TITLE
Error in Ensemble restart generation sampling strategy (Test step 3)

### DISCUSSION THREAD

**Discussion by Lise-Li** [NASA-LIS/LISF#1437] [Q&A] [answered]
Hi, LISF community!

I'm trying to generate the ensemble file in step3. And I had the lis_input files and the open loop restart file. But I met the following error:
#Regridding: Applying budget-bilinear interp to input parameter
 [INFO] Done reading NCEP Native greenness min file. 
 Checking/filling mask values for: SHDMIN
 Total number of surface tiles         616
 Total number of tiles for LSM         616
 Total number of tiles for Openwater           0
 Total number of grid points:         616
 [ERR] Ensemble restart generation sampling strategy: not defined
 [ERR] options are ..'none' and 'random sampling'
 endrun is being called

I attached my lis_input files, open-loop restart file, ldt.config and ldtlog.0000. I was following the public_walkthrough step3. Is there something wrong with one of my files?

[step3 error.zip](https://github.com/NASA-LIS/LISF/files/13162587/step3.error.zip)

Please kindly let me know how to solve the problem.
Thank you
Lise Li

**Comment by dmocko** [comment, marked as answer]
Hi @Lise-Li 

You may have an older version of the Step 3 testcase.  Can you please download the latest?
https://nasa-lis.github.io/LISF/public_testcase_walkthrough/public_testcase_walkthrough.html#_step_3_ensemble_restart_file_generation_ldt

The testcase has this line in the ldt.config, which you do not:
`Ensemble restart generation sampling strategy: none`

All current testcase files were updated in June 2023.

**Reply by Lise-Li** [reply]
Hi, @dmocko !
Thank you for your help! I download the latest, and my problem has been solved!
